business_rules/operators.py

```diff
@@ -238,6 +238,31 @@ def contains(self, other_value):
             if self._case_insensitive_equal_to(val, other_value):
                 return True
         return False
+    
+    @type_operator(FIELD_SELECT, assert_type_for_arguments=False)
+    def contains_any(self, other_value):
+        if not self.value or not other_value:
+            return False
+        for val in self.value:
```

Comment:
n^2?

Reply:
probably okay but rules engine runs every condition a bunch to see if it qualifies so optimization could be worth